app/src/main/java/com/keylesspalace/tusky/db/AppDatabase.java

```diff
@@ -25,10 +25,11 @@
  * DB version & declare DAO
```

Comment:
Yes I changed schemes multiple times while developing, if you had an earlier dev version it will crash. I will check this doesnt happen when upgrading from Tusky 1.4 -> 1.5 before releasing.